astroquery/sdss/field_names.py

```diff
@@ -46,9 +46,13 @@ def get_field_info(cls, tablename, sqlurl, timeout=conf.timeout):
         try:
             _cached_table_fields[key] = _columns_json_to_table(qryres.json())
         except ValueError:
-            warnings.warn("Field info are not available for this data release",
-                          AstropyUserWarning)
-            _cached_table_fields[key] = Table(names=('name',))
+            if isinstance(qryres, MockResponse):
```

Comment:
this is a really stupid workaround here, I suspect we had better rethink and refactor the logic.